jscl.lisp

```diff
@@ -97,7 +97,9 @@
        ("tools"         :target)
        ("macros"        :target)
        ("exports"       :target)
-       ("methods"       :target)) ))
+       ("methods"       :target)
+       ("describe"      :target))
+    ("load"          :target)))
```

Comment:
Please, make sure indentation is correct. It seems it is broken in a few other places as well.

Reply:
Hmmm